tools/testing/update_slow_tests.py

```diff
@@ -204,5 +213,6 @@ def search_for_open_pr(
         # no existing pr, so make a new one and approve it
         pr_num = make_pr("pytorch/pytorch", params)
         time.sleep(5)
+        add_labels("pytorch/pytorch", pr_num, ["ciflow/slow", "ci-no-td"])
```

Comment:
Alternatively, you could update `titleRegexToLabel` in https://github.com/pytorch/test-infra/blob/main/torchci/lib/bot/autoLabelBot.ts#L19 to add these labels.  I think that would be cleaner than writing another `add_labels` function here

Reply:
I think I'd prefer to leave it here in order to keep everything in one place